package.json

```diff
@@ -75,7 +74,6 @@
     "@web3-onboard/ledger": "2.3.2",
     "@web3-onboard/trezor": "^2.4.2",
     "@web3-onboard/walletconnect": "^2.5.4",
-    "@web3auth/mpc-core-kit": "^1.1.3",
     "blo": "^1.1.1",
     "bn.js": "^5.2.1",
```

Comment:
This dependency was also only used by social login.

Reply:
Removed. Looks like quite a few other package still depend on it.